jena-base/pom.xml

```diff
@@ -50,21 +50,29 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-csv</artifactId>
-    </dependency> 
-    
+    </dependency>
+
     <!-- supports persistent data structures -->
     <dependency>
       <groupId>com.github.andrewoma.dexx</groupId>
       <artifactId>dexx-collections</artifactId>
     </dependency>
-    
+
     <!-- Remove when jena-base is used -->
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
+    </dependency><dependency><groupId>org.apache.jena</groupId>
```

Comment:
Again, why is `jena-shaded-guava` being pulled in twice, with the wrong version, when it is already pulled in above?